api-documentation.raml

```diff
@@ -53,13 +70,79 @@ types:
       payment: true
       date: "2019-05-10T04:18:00Z"
       amount: 85.6
+/clients:
```

Comment:
Should we also create an action to retrieve all loans for a given client? This could be useful to deal with the business rules with more ease.

Reply:
I thought about that and I also think it is a good idea. I'll put those on.